src/Mod/PartDesign/Gui/TaskChamferParameters.cpp

```diff
@@ -328,6 +346,96 @@ void TaskChamferParameters::apply()
     }
 }
 
+void TaskChamferParameters::setupGizmos(ViewProviderDressUp* vp)
+{
+    if (!Gizmos::isEnabled()) {
+        return;
+    }
+
+    gizmos = std::make_unique<Gizmos>();

```

Comment:
shouldn't Gizmos be created in ViewProvider and then `attachGizmo` would simply be used to add gizmos?